File: JAVA/Numeric/fibonacci.java
import java.util.*;

public class fibonacci {

  public static void main(String[] args) {
    
    int n = 0;
    int m = 1;
    int f = 0;
    int sum = 0;

    System.out.println(n);
    System.out.println(m);
    for (int i = 0; i < 5; i++) {

      f = n + m; 
      sum += f;
      n = m;
      m = f;
      System.out.println(f);
    }

    System.out.println("Sum is :-"+sum);

  }
}

File: Javascript/Javascript dsa coding/String/fibonacci.js
function fibonacci(n) {
  let a = 0;
  let b = 1;

  for (let i = 0; i < n; i++) {
    console.log(a);

    sum = a + b;
    a = b;
    b = sum;
  }
}

let arr = [7, 45, 8, 89, 55, 8];

console.log(fibonacci(10));
